README-UI.md

```diff
@@ -110,10 +110,11 @@ Jest, Bigtest, or both can be run as part of the workflow. The default is to run
 
 ### Sonarcloud
 
-| Input              | Type   | Default   | Description                                                                                                                                                                                                                                 |
-| ------------------ | ------ | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `sonar-sources`    | string | `./src`   | A comma-separated list of directories containing code to analyze (no wildcards or whitespace); see [Sonar's documentation](https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-parameters/#analysis-scope) for more information |
-| `sonar-exclusions` | string | See below | A comma-separated list of wildcards to exclude from analysis. You can use whitespace as needed for formatting.                                                                                                                              |
+| Input              | Type    | Default   | Description                                                                                                                                                                                                                                 |
+| ------------------ | ------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sonar-enabled`    | boolean | `true`    | Whether to run SonarCloud analysis                                                                                                                                                                                                          |
+| `sonar-sources`    | string  | `./src`   | A comma-separated list of directories containing code to analyze (no wildcards or whitespace); see [Sonar's documentation](https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-parameters/#analysis-scope) for more information |
+| `sonar-exclusions` | string  | See below | A comma-separated list of wildcards to exclude from analysis. You can use whitespace as needed for formatting.                                                                                                                              |
```

Comment:
Are you confident about the whitespace here? [https://github.com/folio-org/stripes-components/pull/2320/commits shows ](https://github.com/folio-org/stripes-components/pull/2320/commits/364ae5eb5fa24fe5151d5320ad70677c4a18f851) begs to differ.

Reply:
@zburke yep, there's some `tr` magic specifically to handle it: https://github.com/folio-org/.github/blob/fef0a160cf04eeba38e96cde258b73c9a6b42cf1/.github/workflows/ui-sonarcloud.yml#L62

Reply:
nvm, the whitespace handling is only for sonar-exclusions. sonar-sources is only 1-3 directories normally, whereas the exclusions list is massive. We can support both with whitespace, if needed